ragen/env/search/utils/reward_score/qa_rouge.py

```
@@ -0,0 +1,148 @@
+        scores = []
+        for ref in reference:
+            if ref:  # Skip empty references
+                score = scorer.score(ref, prediction)
+                scores.append(score[rouge_type].fmeasure)
+        return max(scores) if scores else 0.0
+    else:
+        score = scorer.score(reference, prediction)
+        return score[rouge_type].fmeasure
+
+
+def compute_score_rouge(solution_str: str, ground_truth: Dict, method: str = 'rougeL', format_score: float = 0.0, score: float = 1.0, return_dict: bool = False) -> Union[float, Dict]:
```

Ruff (0.14.10)

Boolean-typed positional argument in function definition

(FBT001)